recipes/GIGATL/recipe.py

```diff
@@ -87,8 +87,8 @@ def create_recipes(datatype, season, pattern):
 surf_pattern = "ftp://eftp.ifremer.fr/SWOT/SURF/gigatl1_1h_tides_surf_avg_{reg:1d}_{yymmdd}.nc"
 
 int_pattern = (
-        "sftpgula@draco.univ-brest.fr:/GIGATL1/SWOT/gigatl1_1h_tides_region_{reg:02d}_{yymmdd}.nc"
-    )
+    "sftpgula@draco.univ-brest.fr:/GIGATL1/SWOT/gigatl1_1h_tides_region_{reg:02d}_{yymmdd}.nc"
```

Comment:
@cisaacstern Can you update the ftp link to: `ftp://eftp.ifremer.fr/SWOT/3D/` ?

Reply:
The credentials are the same for surface data.

Reply:
Thanks for this, @roxyboy! I'd been wondering why I couldn't connect to the int data source file server. Turns out I had the wrong path. :smile: Pushed this change and I am now caching these files to build the GIGATL int zarrs.